src/Serilog/Serilog.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Simple .NET logging with fully-structured events</Description>
+    <VersionPrefix>2.4.1</VersionPrefix>
+    <Authors>Serilog Contributors</Authors>
+    <TargetFrameworks>net45;net46;netstandard1.0;netstandard1.3</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyName>Serilog</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
```

Comment:
It was added by `dotnet migrate`. See https://github.com/dotnet/cli/issues/4309 and  https://github.com/dotnet/cli/pull/4311

Without this, I get:
<img width="1312" alt="screen shot 2017-03-08 at 16 50 31" src="https://cloud.githubusercontent.com/assets/570470/23693269/63472e34-041f-11e7-8347-6c9099495d9b.png">